apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.IntegrationTests/MutateRowsTest.cs

```diff
@@ -0,0 +1,230 @@
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Rpc;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Google.Cloud.Bigtable.V2.IntegrationTests
+{
+    [Collection(nameof(BigtableFixture))]
+    public class MutateRowsTest
+    {
+        private readonly BigtableFixture _fixture;
+
+        public MutateRowsTest(BigtableFixture fixture) => _fixture = fixture;
+
+        [Fact]
+        public async Task AddNewRow()
+        {
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            BigtableByteString rowKey = Guid.NewGuid().ToString();
+            var stream = client.MutateRows(
+                tableName,
+                Mutations.CreateEntry(
+                    rowKey,
+                    Mutations.SetCell(
+                        BigtableFixture.ColumnFamily1,
+                        "column_name",
+                        "test12345",
+                        new BigtableVersion(1))));
+
+            var entries = await stream.GetResponseEntries();
+            Assert.True(entries.All(e => e.Status.Code == (int)Code.Ok));
+
+            await BigtableAssert.HasSingleValueAsync(
+                client,
+                tableName,
+                rowKey,
+                BigtableFixture.ColumnFamily1,
+                "column_name",
+                "test12345",
+                new BigtableVersion(1));
+        }
+
+        [Fact]
+        public async Task ModifyRow()
+        {
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            var rowKey = await _fixture.InsertRowAsync(tableName);
+
+            var stream = client.MutateRows(
+                tableName,
+                Mutations.CreateEntry(
+                    rowKey,
+                    Mutations.SetCell(
+                        BigtableFixture.ColumnFamily1,
+                        "modify_row_column",
+                        "new_cell_value",
+                        new BigtableVersion(1))));
+
+            var entries = await stream.GetResponseEntries();
+            Assert.True(entries.All(e => e.Status.Code == (int)Code.Ok));
+
+            await BigtableAssert.HasValueAsync(
+                client,
+                tableName,
+                rowKey,
+                BigtableFixture.ColumnFamily1,
+                "modify_row_column",
+                "new_cell_value",
+                new BigtableVersion(1));
+        }
+
+        [Fact]
+        public async Task DeleteFromColumn()
+        {
+            var tableName = _fixture.TableName;
+            var client = _fixture.TableClient;
+            var rowKey = await _fixture.InsertRowAsync(tableName, version: new BigtableVersion(1));
+            client.MutateRow(
```

Comment:
So just to check, MutateRow is synchronous - whereas MutateRows is somewhat-asynchronous because it returns a stream? Not sure there's anything we can do about that, but it may be worth us highlighting in the docs.

Reply:
Yes, that's correct. I'm curious: what is the issue with that? And I guess more importantly: what do you think should be highlighted in the docs?

Reply:
Well, one might assume that the two behave very similarly - that if `client.MutateRow(mutation)` behaves synchronously, then `client.MutateRows(new[] { mutation })` will as well. It's not clear to me whether `MutateRows(...)` makes the initial call synchronously (so will throw exceptions if there's anything invalid) or not, for example.

Reply:
Makes sense. I added something, but let me know if you think more is needed.